### ISSUE TITLE
Highlight selected word / variable

### ISSUE THREAD

**Issue by coeci** [atom/atom#10773]
As it is in Notepad++ for example, it would be great if a word or variable would be highlighted if it is selected (double clicked).

Of course, there is a package [https://atom.io/packages/highlight-selected] for that, but why not integrate this function as default? There is no other action, which would be blocked by that.

**Comment by 50Wliu** [comment]
This is a duplicate of atom/find-and-replace#138 - feel free to subscribe there for updates.
